readme.md

```diff
@@ -0,0 +1,30 @@
+# OlympusExtra
+
+OlympusExtra is a Hades DLC mod which adds new gods and boons to the Hades game. This includes unique effects, new god curses, duo boons, Thesus's wrath calls, and more. 
```

Comment:
Thesus's wrath is in scope, but hasn't  been tested. I think what you meant is God's wrath. (for the shout)